internal/utility/severity/severity_test.go

```diff
@@ -0,0 +1,93 @@
+package severity_test
+
+import (
+	"math"
+	"testing"
+
+	"github.com/google/osv-scanner/internal/utility/severity"
+	"github.com/google/osv-scanner/pkg/models"
+)
+
+func TestSeverity_CalculateScore(t *testing.T) {
+	t.Parallel()
+
+	type result struct {
+		score  float64
+		rating string
+	}
+	tests := []struct {
+		name string
+		sev  models.Severity
+		want result
+	}{
+		{
+			name: "Empty Severity Type",
+			sev:  models.Severity{},
+			want: result{
+				score:  -1,
+				rating: "UNKNOWN",
+			},
+		},
+		{
+			name: "CVSS v2.0",
+			sev: models.Severity{
+				Type:  models.SeverityCVSSV2,
+				Score: "AV:L/AC:M/Au:N/C:N/I:P/A:C/E:H/RL:U/RC:C/CDP:LM/TD:M/CR:L/IR:M/AR:H",
```

Comment:
just curious - how these strings are generated?

Reply:
I clicked through a bunch of options on these calculators: [v2.0](https://nvd.nist.gov/vuln-metrics/cvss/v2-calculator), [v3.0](https://www.first.org/cvss/calculator/3.0), [v3.1](https://www.first.org/cvss/calculator/3.1), [v4.0](https://www.first.org/cvss/calculator/4.0)